deploy/mariadb-cluster/templates/_helpers.tpl

```diff
@@ -0,0 +1,4 @@
+{{- define "mariadb-cluster.replicas" }}
+{{- if eq .Values.mode "standalone" }}
+{{- 1 }}
+{{- end -}}
```

Comment:
'defined' is not closed properly. add an extra `{{- end -}}` at the end of file.